topovis/topologytoolkit/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 # Boost is required by TTK
 find_package(Boost REQUIRED)
 
+#TBB is required for parallel STL used in separatrix relaxation processor
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+    set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/cmake/")
```

Comment:
what about the other compilers? Will this not cause issues with Clang or MSVC?

Reply:
This seems to be very GCC specific. I have GCC 9.2 on my system, here parallel STL code needs TBB for linking even though it compiles fine without TBB. For MSVC, as @petersteneteg told me, TBB is not needed. For clang, I guess it's not needed because it builds without TBB on jenkins. In previous commit, I had TBB required in both GCC and clang, but then the build failed on jenkins as it doesn't have TBB installed.

This fix was done by @petersteneteg on my system, I have just made the PR :)

Reply:
@martinfalk we only add it for GNU ie gcc